docs/content/guide/di.ngdoc

```diff
@@ -283,9 +283,12 @@ Here is an example of using the injector service:
 // Provide the wiring information in a module
 var myModule = angular.module('myModule', []);
 ```
+Notice that `myModule` is independent as array is empty here. 
+Dependent is always need dependency object to be injected inside it.
 
 Teach the injector how to build a `greeter` service. Notice that `greeter` is dependent on the
 `$window` service. The `greeter` service is an object that contains a `greet` method.
+so, injector now injects `$window` dependency inside `greeter` service.
```

Comment:
I don't feel that this sentence makes sense. If anything it should appear before the previous sentence and should be something more like

> Notice that `greeter` is dependent on the `$window` service, which will be provided by the injector when it is instantiated.